docs-source/content/userguide/managing-domains/domain-lifecycle/introspection.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Domain introspection"
+date: 2020-07-07T08:14:51-05:00
+draft: true
+weight: 5
+description: "This document describes domain introspection in the Oracle WebLogic Server in Kubernetes environment."
+---
+
+
+This document describes domain introspection, when it occurs automatically, and how and when to trigger additional introspections of the domain configuration in the Oracle WebLogic Server in Kubernetes environment.
+
+In order to manage the operation of WebLogic domains in Kubernetes, the Oracle WebLogic Kubernetes Operator analyzes the WebLogic
+domain configuration using an "introspection" job. This job must successfully complete before the operator will begin to start WebLogic Server instances. Because each of the
```

Comment:
Consider working in a mention that the job is named `DOMAIN_UID-introspect-domain-job` and that it runs in the same namespace as the Domain.